simde/x86/avx512f.h

```diff
@@ -274,7 +277,7 @@ typedef union {
 #if !defined(SIMDE_AVX512F_NATIVE) && defined(SIMDE_ENABLE_NATIVE_ALIASES)
   #define SIMDE_AVX512F_ENABLE_NATIVE_ALIASES
   typedef simde__m512 __m512;
-  typedef simde__m512i __m512i;
+  //typedef simde__m512i __m512i;
```

Comment:
?

Reply:
I got an error about about conflicting definitions that this fixed.

The AVX512* includes are very confusing to me